ComposureWatch/src/Components/TitleSection.jsx

```diff
@@ -1,12 +1,15 @@
 const TitleSection = () => {
   return (
-    <div>
-      <div className="z-[-1] absolute top-0 left-0 right-0 bottom-0 overflow-hidden">
-        <video autoPlay loop muted className="object-fill w-full h-full">
+    <div className="h-[90vh] min-h-[600px] max-h-[1000px] flex flex-col justify-center items-center relative">
+      <div
+        id="video-container"
+        className="align-middle w-full h-full absolute top-0 z-[-1]"
+      >
+        <video autoPlay loop muted className="object-cover w-full h-full">
           <source src="src/video/zenHighlight.mp4" type="video/mp4" />
```

Comment:
will you need to import the video src here as well? It might break when you build otherwise.